frame/staking/src/lib.rs

```diff
@@ -1281,7 +1289,7 @@ decl_module! {
 				total: value,
 				active: value,
 				unlocking: vec![],
-				last_reward: Self::current_era(),
+				claimed_rewards: vec![],
```

Comment:
Can the following scenario happen ?
* A validator have produced some blocks,
* the validator claims some reward,
* the validator unbond himself
* the validator bond himself
* the validator claim its reward again as claimed_reward is empty.

anyway you can say that all era in `[last_payout_era, active_era]` are claimed

Reply:
ah good catch, will update

Reply:
fixed and added tests